Rocket.Chat/Controllers/Auth/AuthViewController+LoginServices.swift

```diff
@@ -0,0 +1,176 @@
+//
```

Comment:
Following the ObjC categories style? 🤓

Reply:
What do you think? 🤔

Reply:
@cardoso We're not using this approach on the other extensions... I like it, but then I think we would have to change the other files as well. 🙃